src/rest-api/endpoints/management-endpoint/migration-endpoint.md

```diff
@@ -0,0 +1,215 @@
+        "running": false
+      },
+      "steps": []
+    },
+    ...and so on...
+  ]
+}
+```
+
+## Get Migration Object
+
+```
+GET /management/migration/MIGRATION_ID
+```
+
+### Path Parameters
+
+| Parameter Name   | Type       | Description      |
+| ---------------- | ---------- | ---------------- |
+| **MIGRATION_ID** | **string** | The migration id.|
+
+If no `MIGRATION_ID` is passed, returns all migration objects, see [Get All Migrations](#get-all-migration-objects).
+
+### Response
+
+If successful, returns a JSON representation of the requested migration object.
+
+### Status Codes
+
+- 200 _OK_ - Success.
+- 404 _Not Found_ - Migration with the given `MIGRATION_ID` does not exist.
+
+### Sample
+
+```curl
+curl -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/management/migration/trash-storage
+```
+
+```json
+{
+  "entity-type": "migration",
+  "id": "trash-storage",
+  "description": "Migration of in the trash storage model",
+  "descriptionLabel": "migration.trash-storage",
+  "status": {
+    "state": "property",
+    "step": null,
+    "startTime": 0,
+    "pingTime": 0,
+    "progressMessage": null,
+    "progressNum": 0,
+    "progressTotal": 0,
+    "running": false
+  },
+  "steps": []
+}
+```
+
+## Probe a Migration Object
+
+```
+POST /management/migration/MIGRATION_ID/probe
+```
+
+### Path Parameters
+
+| Parameter Name   | Type       | Description      |
+| ---------------- | ---------- | ---------------- |
+| **MIGRATION_ID** | **string** | The migration id.|
+
+### Response
+
+If successful, probes the current state of a migration by analyzing persistent data, and sets it as the new current state, then returns a JSON representation of the requested migration object.
+
+### Status Codes
+
+- 200 _OK_ - Success.
+- 404 _Not Found_ - Migration with the given `MIGRATION_ID` does not exist.
+
+### Sample
+
+```curl
+curl -X POST -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/management/migration/trash-storage/probe
+```
+
+```json
+{
+  "entity-type": "migration",
+  "id": "trash-storage",
+  "description": "Migration of in the trash storage model",
+  "descriptionLabel": "migration.trash-storage",
+  "status": {
+    "state": "property",
+    "step": null,
+    "startTime": 0,
+    "pingTime": 0,
+    "progressMessage": null,
+    "progressNum": 0,
+    "progressTotal": 0,
+    "running": false
+  },
+  "steps": []
+}
+```
+
+## Run a Migration
+
+```
+POST /management/migration/MIGRATION_ID/run
+```
+
+### Path Parameters
+
+| Parameter Name   | Type       | Description      |
+| ---------------- | ---------- | ---------------- |
+| **MIGRATION_ID** | **string** | The migration id.|
+
+### Response
+
+If successful, probes the current state of a migration by analyzing persistent data, sets it as the new current state, then runs the migration asynchronously.
+
+### Status Codes
+
+- 202 _Accepted_ - Success.
+- 400 _Bad Request_ - Unknown migration `MIGRATION_ID`.
```

Comment:
Isn't it 404 like for the other calls?

Reply:
OK, but still, I believe that it is clearer for the reader to always have the same phrase, no matter the internal message sent by the Nuxeo server.
Indeed, in the "Get Migration Object" section we have:
```
### Status Codes

- 200 _OK_ - Success.
- 404 _Not Found_ - Migration with the given `MIGRATION_ID` does not exist.
```
Let's use the exact same format for the current section "Run a Migration"